js/passage.js

```diff
@@ -1,4 +1,127 @@
-const passage = Object.freeze([
+export class Passage {
+  constructor(verses) {
+    this._verses = verses;
+  }
+
+  metadata() {
+    let firstVerse = this._verses[0];
+    let lastVerse = this._verses[this._verses.length - 1]
+    let bookChapter = firstVerse.book + ' ' + firstVerse.chapter + ':';
+    if (this._verses.length > 1) {
+      if (firstVerse.chapter === lastVerse.chapter) {
```

Comment:
We could potentially hit an edge case here in the future where a book is crossed, but the chapter is the same, thus leading to the second book not being displayed.